tests/API/OVAL/unittests/test_external_variable.sh

```diff
@@ -9,7 +9,10 @@ stderr=$(mktemp ${name}.err.XXXXXX)
 echo "stderr file: $stderr"
 
 $OSCAP oval eval --results $result --variables $srcdir/external_variables.xml $srcdir/$name.oval.xml 2> $stderr
-[ ! -s $stderr ] && rm $stderr
+# filter out the expected warnings in stderr
+sed -i -E "/^W: oscap:     Referenced variable has no values \(oval:x:var:[13689]\)/d" "$stderr"
```

Comment:
What if the indentation level will change in future? I would put here instead of 6 spaces a regular expression matching at least one space.